test/test_config.cfg

```diff
@@ -1,6 +1,7 @@
 [WOF]
 Network: TEST
 Vocabulary: TESTVocab
+URLPATH: TESTURL
 Menu_Group_Name: TEST OBSERVATIONS
 Service_WSDL: http://not.a.real.url/
```

Comment:
I know you're not changing `Service_WSDL` here, but it seems like this would not be needed anymore, right? It can be constructed automatically from `URLPATH`, like this (assumes WOF 1.1): `URLPATH/soap/cuahsi_1_1/.wsdl`

If that's true, it should be removed from here and handled automatically. But this could be done in a follow up PR if you'd like, so that this PR is not delayed.

Reply:
@emiliom Yea, I think that's possible to do, Service_WSDL can be auto-generated. I can work on that next. Thanks.